prep-css-grid-classes/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Grid Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      width: 100%;
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+      margin-bottom: 1rem;
+
+      .column {
+        width: calc(100% /3);
+        padding-right: 1rem;
+        padding-left: 1rem;
+      }
+
+      img {
+        width: 100%;
+        height: 450px;
+        object-fit: contain;
+      }
```

Comment:
somehow, these two rule-sets are inside of your `.row` rule-set. They should be separate.

```css
.row {

}

.column {

}

img {

}
```